vite.config.ts

```diff
@@ -65,6 +65,7 @@ export default ({ mode }: { mode: 'production' | 'development' | 'test' }) => {
       globals: true,
       environment: 'jsdom',
       setupFiles: ['test/vitest.setup.ts'],
+      css: true,
```

Comment:
2 questions:

1. Should we remove the `mode === 'test'` in the css section of the main config above?
2. Can we use `stable` for `classNameStrategy` so there's less noise in the (future) noise in the snapshots? (https://vitest.dev/config/#css-modules-classnamestrategy)

Reply:
The CSS override is redundant indeed, I've removed it!